### PR TITLE
Release 1.10.0

### PR DESCRIPTION
**Pull request by OleksiiZubko**
Update CHANGELOG and up versions.

Resolves: OLPEDGE-2548

Signed-off-by: Oleksii Zubko <ext-oleksii.zubko@here.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
+## v1.10.0 (14/05/2021)
+
+**olp-sdk-dataservice-write**
+
+- Added the `MultiPartUploadWrapper` class for uploading large files in Browser and Node.js to the
+  Blob V1 and V2 services in chunks.
+- Changed `VersionedLayerClient`, replaced uploading logic for large files by `MultiPartUploadWrapper`.
+- Added abstracted class `BlobData`. The wrapper of data (`Blob`, `Buffer`, `File`, string, and so on)
+  that allows to read a specific range of bytes and get data size.
```
Comment (HalynaDumych) [inline CHANGELOG.md:9]:
```suggestion
- Added the `MultiPartUploadWrapper` class. You can use it to upload large files in a browser and Node.js to the Blob API v1 and v2 services in chunks.
- Changed `VersionedLayerClient`. Now, to upload large files to versioned layers, use the  `MultiPartUploadWrapper` class.
- Added the abstracted `BlobData` class. It is a wrapper of data (`Blob`, `Buffer`, `File`, string, and so on)
  that allows to read a specific range of bytes and get data size.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+
+**olp-sdk-dataservice-api**
+
+- Fixed the issue with incorrect `body` in the function `ObjectApi.startMultipartUploadByKey`.
```
Comment (HalynaDumych) [inline CHANGELOG.md:13]:
```suggestion
- Fixed the issue with the incorrect `body` parameter in the `ObjectApi.startMultipartUploadByKey` function.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+
+**olp-sdk-core**
+
+- Added `export * as fs from "fs";` to the Node.js index file and changed typings settings from `index.web` to `index`.
```
Comment (HalynaDumych) [inline CHANGELOG.md:17]:
```suggestion
- Added `export * as fs from "fs";` to the Node.js index file and changed typing settings from `index.web` to `index`.
```

**Review by HalynaDumych — CHANGES_REQUESTED** [review]

**Review by HalynaDumych — APPROVED** [review]

**Review by andescu — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (HalynaDumych) [reply CHANGELOG.md:9]: Please address this comment.

File: CHANGELOG.md (hunk 2 above)
Comment (HalynaDumych) [reply CHANGELOG.md:13]: Please address this comment.

File: CHANGELOG.md (hunk 3 above)
Comment (HalynaDumych) [reply CHANGELOG.md:17]: Please address this comment.

**Review by HalynaDumych — CHANGES_REQUESTED** [review]

**Review by HalynaDumych — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,28 @@
...
+- Added the abstracted `BlobData` class. It is a wrapper of data (`Blob`, `Buffer`, `File`, string, and so on)
+  that allows to read a specific range of bytes and get data size.
+
+**olp-sdk-dataservice-api**
+
+- Fixed the issue with the incorrect `body` parameter in the `ObjectApi.startMultipartUploadByKey` function.
+
+**olp-sdk-core**
+
+- Added `export * as fs from "fs";` to the Node.js index file and changed typing settings from `index.web` to `index`.
```
Comment (andescu) [inline CHANGELOG.md:16]: Probably it is enough to say: `Added export for fs symbols to the Node.js index file....`

**Review by andescu — APPROVED** [review]

**Pull request merged**